<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QMainWindow" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>4078</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>20000</width>
    <height>20000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>1366</width>
     <height>768</height>
    </size>
   </property>
   <widget class="Phonon::VideoPlayer" name="videoPlayer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1150</width>
      <height>650</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1366</width>
      <height>768</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>0</y>
      <width>216</width>
      <height>650</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="button_bar" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>650</y>
      <width>1366</width>
      <height>100</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <widget class="Phonon::VolumeSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_stop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/media-playback-stop.png</normaloff>../icons/media-playback-stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_play">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/media-playback-start.png</normaloff>../icons/media-playback-start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_pause">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../icons/media-playback-pause.png</normaloff>../icons/media-playback-pause.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="Phonon::SeekSlider" name="seekSlider">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>810</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_open">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>32</y>
       <width>165</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>32</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Now Playing:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_prev">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>32</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/media-seek-backward.png</normaloff>../icons/media-seek-backward.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_next">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>32</y>
       <width>71</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/media-seek-forward.png</normaloff>../icons/media-seek-forward.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_maximize">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>32</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/view-fullscreen.png</normaloff>../icons/view-fullscreen.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="lab_video_tot">
     <property name="geometry">
      <rect>
       <x>1220</x>
       <y>5</y>
       <width>70</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lab_video_cur">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>5</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_random">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>32</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/transform-rotate.png</normaloff>../icons/transform-rotate.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_equalizer">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>32</y>
       <width>30</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../icons/view-media-equalizer.png</normaloff>../icons/view-media-equalizer.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4078</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_Playlist"/>
    <addaction name="actionOpen_Playlist"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../icons/application-exit.png</normaloff>../icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../icons/dialog-information.png</normaloff>../icons/dialog-information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_Playlist">
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
  <action name="actionOpen_Playlist">
   <property name="text">
    <string>Open Playlist</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>../icons/system-help.png</normaloff>../icons/system-help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
